simp_le.py

```diff
@@ -997,6 +997,10 @@ def create_parser():
         '--reuse_key', action='store_true', default=False,
         help='Reuse private key if it was previously persisted.',
     )
+    io_group.add_argument(
+        '--use_alt_chain', type=int, default=0, metavar='CHAIN_NO',
+        help='If non-zero, then use alternative certificate chain number.',
```

Comment:
I think this could do with a link to relevant documentation; I suspect many users would have absolutely no idea what this does or why you would use it.